test/common.py

```diff
@@ -34,6 +34,12 @@
 import sockeye.utils
 from sockeye.evaluate import raw_corpus_bleu, raw_corpus_chrf
 
+try:
```

Comment:
why this?

Reply:
Initially the code in`test/common_image_captioning.py`  was part of `test/common.py`. So, I forgot to remove this. I will remove it.